selfdrive/car/toyota/interface.py

```diff
@@ -91,7 +91,7 @@ def get_params(candidate, fingerprint):
       ret.steerKpV, ret.steerKiV = [[0.45], [0.015]]
       ret.steerKf = 0.0000675   # full torque for 10 deg at 80mph means 0.00007818594
       # TODO: Prius seem to have very laggy actuators. Understand if it is lag or hysteresis
-      ret.steerActuatorDelay = 0.40
+      ret.steerActuatorDelay = 0.43
```

Comment:
Damn, that thing is slow as fuck :) I think @pd0wm has a way to measure exact delay

Reply:
Doing a lot of tests, so not necessarily a final value.